packages/ui-new/src/utils/UiHelperUtil.ts

```diff
@@ -16,6 +16,24 @@ export const UiHelperUtil = {
     return new Intl.DateTimeFormat('en-US', { month: 'short', day: 'numeric' }).format(date)
   },
 
+  formatCurrency(amount: number | string = 0, options: Intl.NumberFormatOptions = {}) {
+    const numericAmount = Number(amount)
```

Comment:
Should we cast it to number? What if the number is comes float? and when there are many floating numbers, isn't this will remove them?

Reply:
Okay seems like not yeah